CMakeLists.txt

```diff
@@ -1,20 +1,30 @@
 cmake_minimum_required(VERSION 2.8.3)
+set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
We need a better way of doing this. It's going to be incredibly painful hunting down and destroying all of these as soon as we move to the next version of ros with a gcc that uses c++14 instead.